vision_datasets/commands/utils.py

```diff
@@ -1,17 +1,91 @@
 import base64
 import io
+import locale
 import logging
 import os
-
+import pathlib
+import json
+from typing import Union
+from vision_datasets import DatasetTypes, DatasetManifest, Usages
 from vision_datasets.common.image_loader import PILImageLoader
+from vision_datasets.common.util import FileReader
+
+
+def set_up_cmd_logger(name):
+    logger = logging.getLogger(name)
+    logging.basicConfig(level=logging.INFO)
 
-logger = logging.getLogger(__name__)
-logging.basicConfig(level=logging.INFO)
+    return logger
+
+
+logger = set_up_cmd_logger(__name__)
 
 TSV_FORMAT_LTRB = 'ltrb'
 TSV_FORMAT_LTWH_NORM = 'ltwh-normalized'
 
 
+class Base64Utils:
+    def b64_str_to_pil(img_b64_str: str):
+        assert img_b64_str
+
+        return PILImageLoader.load_from_stream(io.BytesIO(base64.b64decode(img_b64_str)))
+
+    def file_to_b64_str(filepath: pathlib.Path):
+        fr = FileReader()
+        with fr.open(filepath.as_posix(), "rb") as file_in:
+            return base64.b64encode(file_in.read()).decode('utf-8')
+
+    def b64_str_to_file(b64_str: str, file_name: Union[pathlib.Path, str]):
```

Comment:
@shonohs this directly dumps the b64 string into file, to avoid using PIL inbetween which alters the file more or less